moped-editor/src/components/GridTable/FiltersChips.js

```diff
@@ -129,64 +129,60 @@ const FiltersChips = ({
 
   return (
     <Box className={classes.filtersList}>
-      <Typography className={classes.filtersText} component="span">
```

Comment:
Really at the root of all this is dropping this Typography component and reworking the flex-box stuff. I compared the UI from production and this deploy preview side by side, and I am unable to detect any regression -- beyond the chips flowing now as expected.

Reply:
awesome! thanks for updating - i don't see a regression either. Before merge, could you delete the `filtersText` key-value pair in that starts on line 19 since it isn't used anymore. One less style to maintain! 🙏 🙌